### PR TITLE
Delete csv files

### PR DESCRIPTION
**Pull request by luqmannablan**
testing and delete files functionality

### CODE REVIEW

File: config/credentials.yml.enc [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+IcxZIBJFWtOcrbOS02axHgZNdr15xaEnE7QIAX5xIq9rlmaZ4qYpWfTe2If3jYPTFvZ0WXLireI0ZdO6yw2TpEdyzoNLuI0w/qgtdcGkXgDQEMlZoyW4eUrEpK5uwfHJfPi1IW8lzk3m3BVCOu1P9jruzXhx0C1D5PW9tSAUYsujFYuJXfkT6vlTstMdPQXEcPqRL33a8ZkA/Hu9dIcfu1Vr5eqfgEPiCAoCpAW6MfUylshCeeNQlIrVM5A4GbbfRjG75pLe3pC8oIeFCF9f1xe6E6mYlK4DH2YIvqA6WOlN5VkCO1FKrwauB0BAvS4qzwbJ3pfFgB9D0KZ+D+xOefkUeYKstvUxh6tREbQQDtO8/tm4D6a/vo1m3WXQY6iADCQ8MmbjvZBI3Vk4yhLrJZmjaIk7Oei9FFw3nMBAqyipvFehN49gqxO8RdYMaIuinQNkDjzs33ce5wC927fsCE2feyAh4+YLV7WlRvKYD6W4m+aTZ+pn5OmL2vA+gLgpsBniJz8LUtFYtCLW7DqnEMVDB00xKRBMCLlhizlqWEPgo+aVnXzAlvHus9u4GX2f42jUWMXpws00GzJd701I85xz3RgXe1uN--Ig2CgkNM70yjQHZm--/DT9mmTCYPoiXBrhqbWV7g==
```
Comment (haridher-tamu) [inline config/credentials.yml.enc:1]: Don't edit this file, please revert this

File: config/routes.rb [hunk 2]
```diff
@@ -16,6 +16,21 @@
   get "sessions/logout"
   get "sessions/omniauth"
 
-  resources :admins
+  #Route for deleting CSV
+delete "welcome/delete_csv_files", to: "welcome#delete_csv_files", as: "delete_csv_files"
```
Comment (haridher-tamu) [inline config/routes.rb:20]: Run rubocop -a once. Git is giving those warnings.

**Comment by haridher-tamu** [comment]
Besides the 2 minor comments, LGTM. Will approve after that.

**Review by amohanty03 — CHANGES_REQUESTED** [review]
Please look at Hari's comment.

**Pull request closed without merging**